anxcloud/errors.go

```diff
@@ -0,0 +1,8 @@
+package anxcloud
+
+import "errors"
+
+var (
+	// ErrOperationNotSupported is returned when an operation is not supported
```

Comment:
I just love comments that bring absolutely no new info :upside_down_face: 

(nothing to change here, I know the linter enforces it and I don't see anything to add)